go.mod

```diff
@@ -5,21 +5,32 @@ require (
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
 	github.com/beltran/gohive v1.0.0 // indirect
 	github.com/c-bata/go-prompt v0.0.0-20190826134812-0f95e1d1de2e
+	github.com/cosiner/argv v0.0.1 // indirect
```

Comment:
Did you ran `go get ./...` before commiting? we are using gomod now, and `go get` will try to update all dependencies.

Reply:
I ran `go test ./...` which I thought is the reason for this.